### PR TITLE
Approve design updates

### PR DESCRIPTION
**Pull request by rickycodes**
**Description**

Design updates (from here: https://github.com/MetaMask/metamask-mobile/pull/1390#issuecomment-594996341) also included in this PR are the margin/alignment from that same PR.

**Changes to the Adjust Transaction fee Modal:**

** Note: the leading white space seen here will be handled by: https://github.com/MetaMask/metamask-mobile/pull/1454

**Changes to Reset account Modal and buttons:**

**Changes to Transaction Fee lockup for Confirm Transaction Edit:**

** Note: there were some additional changes to be made to this area, but I've opted to do them in a separate PR since this one is already quite large... Discussed with @cjeria and this should be good enough to prevent the text from overflowing on smaller screens.

### CODE REVIEW

File: app/components/UI/TransactionEdit/index.js [hunk 1]
```diff
@@ -427,10 +428,14 @@ class TransactionEdit extends PureComponent {
...
 						{!paymentChannelTransaction && (
 							<View style={[styles.formRow, styles.row, styles.notAbsolute]}>
-								<View style={styles.label}>
+								<View>
 									<Text style={styles.labelText}>{strings('transaction.gas_fee')}:</Text>
 									{gasError ? <Text style={styles.error}>{gasError}</Text> : null}
 								</View>
+							</View>
+						)}
+						{!paymentChannelTransaction && (
```
Comment (danjm-temp) [inline app/components/UI/TransactionEdit/index.js]: Is this second `{!paymentChannelTransaction && (` needed here? Can the new view just be included in the above expression?

File: app/components/Views/SendFlow/CustomGas/index.js [hunk 2]
```diff
@@ -29,20 +29,16 @@ const styles = StyleSheet.create({
...
 		padding: 12,
 		borderWidth: 1,
-		borderRadius: 6,
 		borderColor: colors.grey100,
-		marginVertical: 16
-	},
-	selectorCenter: {
-		marginHorizontal: 8
+		marginTop: -1,
+		zIndex: 0
```
Comment (danjm-temp) [inline app/components/Views/SendFlow/CustomGas/index.js]: Is specifying z-index of 0 necessary?

**Comment by danjm-temp** [comment]
I reviewed and this looks good. A couple small comments, but nothing to block over.

@ibrahimtaveras00 this is ready for QA

**Review by ibrahimtaveras00 — APPROVED** [review]
Updates look good on various screen sizes on both OS's, QA Passed 👍

**Comment by cjeria** [comment]
Where did we leave off with the font inconsistencies? I see Roboto still being used here. Our new font should be CircularXx.

**Pull request merged**